### DISCUSSION TITLE
Add a new feature flag for media players which support enqueuing media

### DISCUSSION THREAD

**Discussion by emontnemery** [home-assistant/architecture#940] [Entity Models] [answered]
### Suggestion:
Add a new feature flag `MediaPlayerEntityFeature.ENQUEUE` for media players which support enqueuing media

### Rationale:
Support for enqueing media when calling the `media_player.play_media` is not mandatory, and only supported by some integrations. By adding a feature flag, it's clear if the media players supports enqueueing

**Comment by frenck** [comment, marked as answer]
Do it! 👍 

../Frenck

**Reply by frenck** [reply]
implemented in https://github.com/home-assistant/core/pull/95905
